openflow13/match.go

```diff
@@ -1482,6 +1483,40 @@ func NewTunnelIpv6DstField(tunnelIpv6Dst net.IP, tunnelIpv6DstMask *net.IP) *Mat
 	return f
 }
 
+type TtlField struct {
+	Ttl uint8
+}
+
+func (m *TtlField) Len() uint16 {
+	return 1
+}
+
+func (m *TtlField) MarshalBinary() (data []byte, err error) {
+	data = make([]byte, 1)
+	data[0] = m.Ttl
+	return
+}
+
+func (m *TtlField) UnmarshalBinary(data []byte) error {
+	m.Ttl = data[0]
```

Comment:
Better to check the length of `data` before accessing the slice with a given index.

Reply:
Done. Has added the data length check.